decidim-elections/spec/commands/decidim/votings/census/admin/create_dataset_spec.rb

```diff
@@ -32,6 +32,16 @@ module Decidim::Votings::Census::Admin
       expect(subject.call).to broadcast(:ok)
     end
 
+    # rubocop:disable Lint/AmbiguousBlockAssociation, RSpec/IteratedExpectation
+    it "enqueues a job for processing the dataset and strips the data from whitespaces" do
+      expect { subject.call }.to have_enqueued_job(CreateDatumJob).at_least(4).times.with { |_user, _dataset, row|
+        row.each do |field|
```

Comment:
⚠️ Instead of browsing all the fields and check the strip, it would be enough to search for an specific field value with an space in the demo file and check that it's value was fixed. That way, you won't need to disable `RSpec/IteratedExpectation` the linter rule.